go/vt/vtgate/evalengine/api_types.go

```diff
@@ -176,16 +175,10 @@ func ToNative(v sqltypes.Value) (any, error) {
 	return out, err
 }
 
-func NormalizeValue(v sqltypes.Value, coll collations.ID) string {
+func NormalizeValue(v sqltypes.Value, coll collations.ID) sqltypes.Value {
```

Comment:
as far as I can see, only tests use this method. is that intentional?

Reply:
@systay Yeah, it's a test only thing. I think we should move it then to the test file. Let me do that.